docs/tfengine/README.md

```diff
@@ -14,7 +14,7 @@ used to generate
 specific to your Google Cloud organization and structure.
 
 Users typically pick from a set of
-[recipes](../../templates/tfengine/schemas) which implement a template for one
+[recipes](../../templates/tfengine/recipes) which implement a template for one
```

Comment:
Ah I remember now what I wanted to do. I was going to point this to docs/tfengine/schemas. WDYT is a better link?

Reply:
You have another link in the next sentences that is refering to the md folder.